### ISSUE TITLE
How to select a single position for atoms with multiple alternate positions listed

### ISSUE THREAD

**Issue by fabiooliveira-at-uib** [openmm/pdbfixer#205]
Hello!
I don't seem to find the option in the python API to select single position for atoms with multiple alternate positions listed.
How should I go about doing it?
Thank you!

**Comment by jchodera** [comment]
Again, there isn't an easy way to do this with the pdbfixer API right now, but this would make an excellent feature addition!

**Comment by fabiooliveira-at-uib** [comment]
Yes, it would be great if that feature could be added. Thank you for the reply!

**Comment by peastman** [comment]
I'm reopening this.  It's a feature request!
